modules/build-config-capability.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// *  installing/cluster-capabilities.adoc
+
+:_content-type: REFERENCE
+[id="build-config-capability_{context}"]
+= Build capability
```

Comment:
Should we provide some minimal description of what the Build API consists of? E.g. the same few lines as in https://github.com/openshift/api/blob/master/config/v1/types_cluster_version.go#L318:
```
Build API is responsible for watching the Build API objects and managing their lifecycle. The following resources are part of the API: builds, buildconfigs.
```